llvm/test/CodeGen/SPIRV/debug-info/basic-global-di.ll

```diff
@@ -0,0 +1,129 @@
+; RUN: llc -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s --check-prefix=CHECK-SPIRV
```

Comment:
If we are not checking multiple llc runs (e.g. with different arguments) then we don't have to use the `--check-prefix=CHECK-SPIRV` flag at all. You can just use `CHECK` or `CHECK-DAG` commands.

Reply:
Done